gamestonk_terminal/stocks/comparison_analysis/ca_controller.py

```diff
@@ -89,61 +96,93 @@ def __init__(
-
-        print("\nComparison Analysis Mode:")
-        print("   cls           clear screen")
-        print("   ?/help        show this menu again")
-        print("   q             quit this menu, and shows back to main menu")
-        print("   quit          quit to abandon program")
-        print("")
-        print("   get           get similar companies")
-        print("   select        select similar companies")
-        print("")
-        print("   historical    historical price data comparison")
-        print("   hcorr         historical price correlation")
-        print("   income        income financials comparison")
-        print("   balance       balance financials comparison")
-        print("   cashflow      cashflow comparison")
-        print("   sentiment     sentiment analysis comparison")
-        print("   scorr         sentiment correlation")
-        print("")
-        print("   overview      brief overview comparison")
-        print("   valuation     brief valuation comparison")
-        print("   financial     brief financial comparison")
-        print("   ownership     brief ownership comparison")
-        print("   performance   brief performance comparison")
-        print("   technical     brief technical comparison")
-        print("")
-
-        if self.similar:
-            print(">  po          portfolio optimization for selected tickers")
-            print("")
-        return
+            stock_str = f"\n{s_intraday} Stock: {self.ticker}"
+        help_str = f"""
+https://github.com/GamestonkTerminal/GamestonkTerminal/tree/main/gamestonk_terminal/stocks/comparison_analysis
+
+{stock_str}
+Similar Companies: {', '.join(self.similar) or None}
+
+Comparison Analysis:
+    cls           clear screen
+    ?/help        show this menu again
+    q             quit this menu, and shows back to main menu
+    quit          quit to abandon program
+
+    load          load new base ticker
+    get           get similar companies
+    add           add more companies to current selected (max 10 total)
+    select        reset and select similar companies
+
+Custom Machine Learning Comparisons:
+    tsne          run TSNE on all SP500 stocks and returns 10 closest tickers
+Yahoo Finance:
+    historical    historical price data comparison
+    hcorr         historical price correlation
+Market Watch:
+    income        income financials comparison
+    balance       balance financials comparison
+    cashflow      cashflow comparison
+Finbrain:
+    sentiment     sentiment analysis comparison
+    scorr         sentiment correlation
+Finviz:
+    overview      brief overview comparison
+    valuation     brief valuation comparison
+    financial     brief financial comparison
+    ownership     brief ownership comparison
+    performance   brief performance comparison
+    technical     brief technical comparison
+
+    >po          portfolio optimization for selected tickers
```

Comment:
the > should be at the start of the line. And the portfolio should be aligned with above ones